techlibs/quicklogic/ap2_lutdefs.txt

```diff
@@ -0,0 +1,5 @@
+1 1.00 1.00
```

Comment:
what does 3 columns entries stand for?

Reply:
So these are ABC LUT definitions read by it with `read_lut` commands. The columns are: number of inputs, area, propagation delay. Normally Yosys automatically generates them when you invoke `abc -lut/-luts`. So I managed to extract exactly those and put them here.

During working on optimizing Yosys output size I played with values of these fields. But it didn't yield in any significant gains.

Actually I'm not sure whether the ones for AP2 and AP3 are correct. These architectures contain LUT4 and LUT4/LUT5 respectively only and here ABC will emit also LUTs 1-3. But I didn't want to break anything so I left the LUT definitions as they were.